### ISSUE TITLE
Error 404 On External Repo

### ISSUE THREAD

**Issue by ThePieMonster** [ncipollo/release-action#182]
**Describe the bug**
When trying to create a release for another repo, a 404 error is generated. The error handling could be more detailed as well.

**To Reproduce**
1. Create a personal access token with the "repo" permission. 
2. Include that token as a secret in the repo.
3. Create a GitHub Action as shown below:

```
    - name: Create Release
      id: create_release
      uses: ncipollo/release-action@v1
      with:
        name: ${{ env.VERSIONNAME }}
        tag: ${{ env.VERSIONNAME }}
        body: ${{ steps.Changelog.outputs.changelog }}
        token: ${{ secrets.REPO_TOKEN }}
        repo: OtherGitHubAccount/repo.name
```

The output is: `Error: Error 404: Not Found`

**Expected behavior**
The release is created in the specified repo.

**Comment by ncipollo** [comment]
The format you are using is a bit off:
- `repo` should be just the name of the repo. Using this project as an example, `repo` would be `release-action`.
- You need to specify owner. Using this project as an example, `owner` would be `ncipollo`.

tldr; It's split across two properties, not one - `owner` and `repo`.

**Comment by ThePieMonster** [comment]
This did the trick. Thanks!

**Comment by benfogiel** [comment]
@ncipollo is it possible to reference a specific branch? So for example, if this repo had a branch named `someBranch` would it be possible to do something like `repo: ncipollo/someBranch`? I've tried this myself and it's not working unfortunately, however, if this feature does exist, I'm assuming I have the incorrect syntax. Thank you in advance for any assistance
